providers/common/provider_err.c

```diff
@@ -152,6 +152,8 @@ static const ERR_STRING_DATA PROV_str_reasons[] = {
     "operation not supported for this keytype"},
     {ERR_PACK(ERR_LIB_PROV, 0, PROV_R_OUTPUT_BUFFER_TOO_SMALL),
     "output buffer too small"},
+    {ERR_PACK(ERR_LIB_PROV, 0, PROV_R_PARENT_CANNOT_GENERATE),
+    "parent cannot generate"},
```

Comment:
If I as a user would be confronted with the error message "parent cannot generate", I would be quite confused.

Reply:
Added "random numbers" to the message.